### DISCUSSION TITLE
Form de entrega de QueMePongo?

### DISCUSSION THREAD

**Discussion by guidoenr** [dds-jv/foro#182] [Consultas generales] [answered]
Que tal, termine el ejercicio de QueMePongo, pero cuando quiero enviarlo al link del sitio web que dice, me redirecciona aca
 
https://aulasvirtuales.frba.utn.edu.ar/enrol/index.php?id=2536

tenia pensado que iba a ser un form como el de la clase pasada de macowins, capaz es diferente, pero de ser asi, esa aula me pide clave de matriculacion que nose cual es.

**Comment by gastonprieto** [comment, marked as answer]
Hola @guidoenr !

Había quedado mal el link, ya lo corregí. Fijate si ahora podes acceder al form
